src/cli/onefuzz/templates/aflpp.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from typing import Dict, List, Optional
+
+from onefuzztypes.enums import OS, ContainerType, StatsFormat, TaskType
+from onefuzztypes.models import NotificationConfig
+from onefuzztypes.primitives import Container, Directory, File
+
+from onefuzz.api import Command
+
+from . import JobHelper
+
+
+class AFL(Command):
```

Comment:
This class isn't used or exposed anywhere yet. FYI, to actually expose it as a subcommand of `template`, it would need to instantiated and [bound to `self`](https://github.com/microsoft/onefuzz/blob/31f4c51b87b542ef958e162f1ff55cb8c3016bc8/src/cli/onefuzz/template.py#L24) in `.template.Template.__init__()`.

Reply:
Also, we'd probably want to name this something like `AFLPlusPlus` to prevent ambiguity in the existing use site linked above.